app/matplotlib.ipynb

```diff
@@ -233,22 +248,35 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
+    "fig, axs = plt.subplots(1, 2, figsize=(10, 5))\n",
     "\n",
-    "# write your code here\n"
+    "axs[0].plot(x, y1, color=\"blue\")\n",
+    "axs[0].set_title(\"Sin Function\")\n",
+    "axs[0].set_xlabel(\"x\")\n",
+    "axs[0].set_ylabel(\"sin(x)\")\n",
+    "\n",
+    "axs[1].plot(x, y2, color=\"red\")\n",
+    "axs[1].set_title(\"Cos Function\")\n",
+    "axs[1].set_xlabel(\"x\")\n",
+    "axs[1].set_ylabel(\"cos(x)\")\n",
+    "\n",
+    "fig.suptitle(\"Sin and Cos Functions\")\n",
+    "\n",
+    "plt.tight_layout()"
```

Comment:
You need to add `plt.show()` at the end of this cell to display the subplots as per the task requirements.